stack.yaml

```diff
@@ -10,6 +10,8 @@ extra-deps:
   - splitmix-0.0.2@sha256:e1c4d1202757d63cd4b4498d5fa600a792f2a315d2fdc6cf772f4679113c1819
   - tree-diff-0.0.2.1@sha256:4bd07bd26091f21710288c1037124bfe19aadb5bdf3df57c8374afaa5f14fef8
   - graphviz-2999.20.0.3
+  - git: https://github.com/kderme/hs-rqlite
```

Comment:
The cabal based CI builds won't like this (adding it to cabal.project will only work with newer cabals). Could you make a new release perhaps?